src/implementation/Server/HTTPServer/HTTPServerImpl.ts

```diff
@@ -146,6 +154,57 @@ export default class HTTPServerImpl {
     );
   }
 
+  async processBulkSubscribeMessage(
```

Comment:
Yeah, it will be still required, as bulk subscribe has git its own processing logic. But, inside it, I was able to re-use process of call back method and retry logic.